README.md

```diff
@@ -74,6 +74,8 @@ Instances of the following are logged as-is using Jackson serialization:
 * Number
 * Boolean
 * String
+* `java.util.Map` where the `key` is `String`
```

Comment:
You want to specify the full class path for the above and for "String"? Also apply the same formatting please.